app/src/main/java/com/example/plantsapp/presentation/ui/tasksfordays/TasksForDaysFragment.kt

```diff
@@ -56,6 +68,24 @@ class TasksForDaysFragment : Fragment(R.layout.fragment_tasks_for_days) {
         }
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.menu_tasks_appbar, menu)
+
+        viewModel.isStatisticsAppInstalled.observe(viewLifecycleOwner) { isInstalled ->
+            menu.findItem(R.id.action_statistics).isVisible = isInstalled
```

Comment:
Also check that this part of code works in isolation (do some changes to LiveData from ViewModel)